### PR TITLE
Akka Streams Integration, codename Alpakka

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: _posts/2016-08-23-intro-alpakka.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Don't hesitate to get involved!
+
+In upcoming blog posts we will describe how to use the `GraphStage` API for building Sinks and Sources to connect to external data sources over various integration protocols. We will show how to handle challenges such as blocking and asynchronous communication. Transformations are also important for integration scenarios and we will illustrate how to implement a streaming XML parser as an example of such encoder/decoder stages.
+
+Akka Streams already has a lot that are useful for integrations. Defining processing pipelines is what the Akka Streams DSL is all about and that is exactly what you need for operating on streaming data that cannot fit in memory as a whole. It handles backpressure in an efficient non-blocking way that prevents out-of-memory errors, which is a typical problem when using unbounded buffering with producers that are faster than consumers.
+
+The following are examples of things that are readily available for building your integrations with Akka Streams today.
```
Comment (ktoso) [inline _posts/2016-08-23-intro-alpakka.md]: `... today (all available with Java and Scala APIs)`?

File: _posts/2016-08-23-intro-alpakka.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+* [Akka Http](http://doc.akka.io/docs/akka/2.4/java/http/index.html) - HTTP client and server components, including support for WebSockets.
+* [Akka Stream Kafka](https://github.com/akka/reactive-kafka) - Connector to Kafka.
+* [http://reactive-streams.org/](Reactive Streams) - Interoperate seamlessly with other Reactive Streams implementations. For example, you can use Akka Streams together with [MongoDB Reactive Streams Java Driver](https://mongodb.github.io/mongo-java-driver-reactivestreams/) for integrating with MongoDB.
```
Comment (johanandren) [inline _posts/2016-08-23-intro-alpakka.md]: Link and title flipped, should be `[Reactive Streams](http://reactive-streams.org/)`

**Comment by johanandren** [comment]
LGTM

File: _posts/2016-08-23-intro-alpakka.md (hunk 1 above)
Comment (patriknw) [reply _posts/2016-08-23-intro-alpakka.md]: ok

**Comment by ktoso** [comment]
LGTM

File: _posts/2016-08-23-intro-alpakka.md (hunk 2 above)
Comment (patriknw) [reply _posts/2016-08-23-intro-alpakka.md]: good catch

**Pull request merged**